Sources/HummingbirdAuth/Sessions/SessionAuthenticator.swift

```diff
@@ -13,28 +13,28 @@
 //===----------------------------------------------------------------------===//
 
 import Hummingbird
+import Logging
 
 public protocol SessionUserRepository: Sendable {
     associatedtype User: Authenticatable
     associatedtype Session: Codable
-    associatedtype Context: RequestContext & AuthRequestContext
 
-    func getUser(from session: Session, context: Context) async throws -> User?
+    func getUser(from session: Session, logger: Logger) async throws -> User?
```

Comment:
Do we never need more context than a Logger? I can imagine sessions being tired to IP address or a country origin